x-pack/plugins/apm/public/components/app/service_overview/service_overview_errors_table/index.tsx

```diff
@@ -147,14 +147,16 @@ export function ServiceOverviewErrorsTable({ serviceName }: Props) {
                 start,
                 end,
                 numBuckets: 20,
-                groupIds: JSON.stringify(
-                  items.map(({ groupId: groupId }) => groupId).sort()
-                ),
                 offset:
                   comparisonEnabled && isTimeComparison(offset)
                     ? offset
                     : undefined,
               },
+              body: {
+                groupIds: JSON.stringify(
+                  items.map(({ groupId: groupId }) => groupId).sort()
```

Comment:
It looks like `items` only contains the current page's group id's. That's fine but different from how this is handled in `error_group_overview/index.tsx`. Can we handle this more consistently?

Reply:
That's why `error_group_overview` uses `ManagedTable` and `service_overview_errors_table` uses `EuiBasicTable`. I personally don't like the `ManagedTable` since it introduces a bug if it is used more than one time on the same page since it updates the table options in the URL (pageIndex, pageSize, sortField and sortDirection).